examples/gocean/eg5/extract/README.md

```diff
@@ -0,0 +1,91 @@
+# PSyclone GOcean PSyData Example 1 - Extraction
+
+**Author** J. Henrichs, Bureau of Meteorology
+
+## Introduction
+
+This is a very simple test that shows how to use the kernel-data extraction
+support in PSyclone. It is a stand-alone program that can be compiled
+and run.
+
+## Compilation
+This example needs two libraries: the gocean infrastructure library
+dl_esm_inf, and a corresponding extraction library. By default
+it will use the infrastructure library in ``../../../../external/dl_esm_inf``
+and automatically compile this library
+(see https://psyclone-dev.readthedocs.io/en/latest/working_practises.html
+for the correct way of checking out all required software) . You can set
+the environment variable ``INF_DIR`` to point to a different directory.
+The NetCDF extraction library in ``../../../../lib/extract/netcdf`` is used
+as default, and will also be automatically compiled. You can set the
+environment variable ``EXTRACT_DIR`` to point to a different library if 
+required. More details on compiling these libraries are in the
+corresponding subdirectories.
+To create and compile the example, modify the Makefile if required
+and type ``make``.
+
+PSyclone is invoked with the script ``extract_transform.py`` which will 
```

Comment:
"PSyclone is supplied with the ``extract_transform.py`` transformation script which ..."

Reply:
Done.